Unifying applications (#14)
greenelab/knowledge-graph-review: content/04.network_applications.md

### Unifying Applications

> Knowledge graphs have been used in numerous amount of biomedical applications.

Reviewer: ```suggestion
Knowledge graphs have been used in many biomedical applications ranging from identifying protein functions [@doi:10.1186/s12859-018-2163-9] to prioritizing cancer genes [@doi:10.1093/bioinformatics/bty148] to recommending safer drugs to patients [@arxiv:1710.05980; @doi:10.1609/aaai.v33i01.33011126].
```

> These applications range from identifying protein functions [@doi:10.1186/s12859-018-2163-9] to prioritizing cancer genes [@doi:10.1093/bioinformatics/bty148] to recommending safer drugs to patients [@arxiv:1710.05980; @doi:10.1609/aaai.v33i01.33011126].

Reviewer: ```suggestion
```

> In this section we discuss various work that uses knowledge graphs for biomedical applications. 

Reviewer: ```suggestion
In this section we discuss how knowledge graphs are being applied in biomedical settings. 
```

> Plus, we emhasize approaches that project knowledge graphs into a low dimensional space using techniques previously discussed (Unifying Techniques).

Reviewer: ```suggestion
We put particular emphasis on an emerging set of techniques: those that project knowledge graphs into a low dimensional space.
```
